objectivec/GPBCodedInputStream.h

```diff
@@ -35,6 +35,34 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+CF_EXTERN_C_BEGIN
+
+/// GPBCodedInputStream exception name.
+extern NSString *const GPBCodedInputStreamException;
+
+/// NSError domain used for GPBCodedInputStream errors.
+extern NSString *const GPBCodedInputStreamErrorDomain;
```

Comment:
I see your point about the exceptions, but adding so many strings (as I did in the second commit) seems wasteful for what is (or should be) code of little use...